src/MovieProfile/MovieProfile.js

```diff
@@ -40,20 +39,13 @@ const MovieProfile = (props) => {
       <article className="profile-container" id={id}>
         <div id="backdrop" className="backdrop" style={styles.header}>
           <div style={styles.contentContainer}>
-            <Link to={`/`} >
-              <button className="go-back btn" onClick={() => props.resetCurrentMovie()}>
-                <i className="fas fa-arrow-left"> </i>
-              </button>
-            </Link>
-            {/* todo ==> make the delete button a redirect? */}
-            <button className="delete btn">
-              <i className="fas fa-times"> </i>
-            </button>
+            <Buttons resetCurrentMovie={props.resetCurrentMovie} />
+
             <div className="content">
               <img className="profile-card" src={poster_path} alt={title}/>
               <div className="wrapper">
                 <span className="ratings">{tomatillos.repeat(Math.floor(average_rating))}</span>
-                <h1 className="movie-title">{title} <span className="year">({release_date})</span></h1>
+                <h1 className="movie-title">{title} <span className="year">(2020)</span></h1>
```

Comment:
is there a reason you hardcoded the year?

Reply:
Yeah, by this point in the components, it seems to lose the context for release_date, because I can't use a split on it at that point. I will probably end up refactoring this into a class component.